src/sections/Meshery/Meshery-integrations/Integration.style.js

```diff
@@ -0,0 +1,43 @@
+import styled from "styled-components";
+
+const Integrations = styled.div`
+  max-width: 1200px;
+  margin: 83px auto;
+
+  .heading {
+    text-align: center;
+
+    h1 {
+      font-size: 3.14rem;
```

Comment:
Why this _fancy_ number?
We have the font size pre-defined, that should be fine for this page too.